docs/docs/30-how-to-guides/20-managing-credentials.md

```diff
@@ -106,6 +106,41 @@ Refer to
 [the advanced section of the installation guide](./10-installing-kargo.md#advanced-installation)
 for more details.
 
+While doing this you must ensure that the Kargo controller has the appropriate
+permissions to read `Secret` resources in these namespaces. This is necessary because,
+as of Kargo v1.0.0, the controller does not have cluster-wide access to `Secret`s. Permissions
+are granted dynamically on a per-Project basis as `Projects` are created.
+
+You can do so by creating your own `RoleBinding`s to permit the Kargo
+ controller(s) to read Secret(s) from each of those namespaces:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: kargo-controller-read-secrets
+  namespace: kargo
```

Comment:
It is probably not ideal to use the `kargo` namespace as a home for shared creds because:

* Controllers would gain access to Kargo's own _internal_ Secrets
* Users authorized to create global creds would require r/w on Secrets in Kargo's own namespace, which would also give them access to Kargo's own _internal_ Secrets

I don't feel like we need to caution too explicitly against this, but for sure, our example shouldn't imply that is viable.

```suggestion
  namespace: kargo-global-creds
```